apps/chat/src/components/Chat/Publish/PublicationHandler/PublicationFilters.tsx

```diff
@@ -27,17 +29,40 @@ interface FilterComponentProps {
   isRulesLoading: boolean;
 }
 
+const showNoRulesLabel = (
+  rulesOnEditLength: number,
+  isNoRulesToDisplay: boolean,
+  publicationModelObject?: { publishToUrl: string },
```

Comment:
Why it needs to be an object if it has only one property and used only for condition?
```suggestion
  publishToUrl?: string,
```